router/helpers_linux.go

```diff
@@ -0,0 +1,37 @@
+package router
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"strings"
+)
+
+func linuxReleaseAttributes() (map[string]string, error) {
+	attrs := map[string]string{}
+	data, err := ioutil.ReadFile("/etc/os-release")
+	if err != nil {
+		return nil, err
+	}
+	s := bufio.NewScanner(bytes.NewReader(data))
+	for s.Scan() {
+		line := s.Text()
+		if len(line) == 0 {
+			continue
+		}
+		line = strings.Replace(line, "\"", "", -1)
+		line = strings.ToLower(line)
```

Comment:
This ToLower is likely going to conflict with looking for NAME and VERSION_ID in the function below

Reply:
In general I'd recommend a regular expression for this kind of parsing just for flexibility as well.. You should be able to find one in the wild that supports the parsing of `NAME="value"` (or `NAME=value`) style lines